### PR TITLE
Java aqa lesson2

### PR DESCRIPTION
**Pull request by LeylaRzazade**

### CODE REVIEW

File: JavaLessons/src/main/java/Lesson2/Method8.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  индексы таких элементов равны, то есть [0][0], [1][1], [2][2], …, [n][n];
+ */
+
+public class Method8 {
+
+    public static void createTwoDimArray(){
+        int[][] myArray = new int[4][4];
+        int i;
+        int j;
+        for (i=0; i<myArray.length; i++){
```
Comment (IrinaPodoliak) [inline JavaLessons/src/main/java/Lesson2/Method8.java:17]: you can avoid initializing you i and j here, it's a common practice to use for cycle in format for(int i = 0; i < array.length; i++)

File: JavaLessons/src/main/java/Lesson2/ShiftArray.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import java.util.Arrays;
+/*
+Написать метод, которому на вход подается одномерный массив и число n (может быть положительным, или отрицательным),
+при этом метод должен сместить все элементы массива на n позиций. Элементы смещаются циклично. Для усложнения задачи
+ нельзя пользоваться вспомогательными массивами. Примеры: [ 1, 2, 3 ] при n = 1 (на один вправо) -> [ 3, 1, 2 ]; [ 3, 5, 6, 1]
+  при n = -2 (на два влево) -> [ 6, 1, 3, 5 ]. При каком n в какую сторону сдвиг можете выбирать сами.
+ */
+public class ShiftArray {
+    public static int[] shiftArray(int[] myArray, int num){
+        int[] myNewArray = new int[myArray.length];
```
Comment (IrinaPodoliak) [inline JavaLessons/src/main/java/Lesson2/ShiftArray.java:11]: working silution, but unfortunately implementation does not correlate to all te requirenments

File: JavaLessons/src/main/java/Lesson2/SwapNumbers.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+/*
+Задать целочисленный массив, состоящий из элементов 0 и 1.
+Например: [ 1, 1, 0, 0, 1, 0, 1, 1, 0, 0 ].
+ С помощью цикла и условия заменить 0 на 1, 1 на 0;
+*/
+
+public class SwapNumbers {
+    public static void swapNumbers() {
+        int[] myArray = new int[]{0, 1, 0, 1, 0, 0, 1};
+        int array_length = myArray.length;
```
Comment (IrinaPodoliak) [inline JavaLessons/src/main/java/Lesson2/SwapNumbers.java:12]: your variable here is used only once and the name is as log as array.length :).  So no need to create it

File: JavaLessons/src/main/java/Lesson2/Method8.java (hunk 1 above)
Comment (LeylaRzazade) [reply JavaLessons/src/main/java/Lesson2/Method8.java:17]: Hello, thank you. Initially, I used in that version, but I had error, I was in a hurry, andended up excluding them from for loop :( Will be noted for future practices.

**Pull request merged**